### PR TITLE
[TU] - basic architecture for TU

### PR DESCRIPTION
**Pull request by Gerroll**
This PR will add the architecture to provide unit test to this projet.

To test:
- launch test.py with python

To add some tests:
- Please add fonction too classes that are named "TestXXX"
- Their will be automaticly add to all test
- To have more depth with what is going on please check the documentation of "unittest"

Good review !

### CODE REVIEW

**Comment by maximegavens** [comment]
Adding test seam to be working ;) great work !

I also found two fail tests when i ran `python test.py`:
`FAIL: test_isModelCreated (naturalLanguageProcessing.TestNlp.TestNLP)`
...
`FAIL: test_predict (naturalLanguageProcessing.TestNlp.TestNLP)`
Is it normal ?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # T-AIA_901
 Voice recognition to return the optimal train route between two destination
 
+# Launch app
+python app.py
```
Comment (maximegavens) [inline README.md:5]:
```
/home/maxime/Project/Epitech/2020-2021/T-AIA_901/venv/lib/python3.7/site-packages/pydub/utils.py:170: RuntimeWarning: Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work
  warn("Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work", RuntimeWarning)
loading : fr_core_news_sm
('No End or invalid instruction', 'Bad Phrase')
('No End or invalid instruction', 'Bad Phrase')
('No End or invalid instruction', 'Bad Phrase')
('No End or invalid instruction', 'Bad Phrase')
('No End or invalid instruction', 'Bad Phrase')
('No End or invalid instruction', 'Bad Phrase')
('No End or invalid instruction', 'Bad Phrase')
('No End or invalid instruction', 'Bad Phrase')
True
True
Could not find PyAudio; check installation
```

Comment (maximegavens) [reply README.md:5]: c'est normal comme affichage ?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+
+# Launch test
+python test.py
```
Comment (maximegavens) [inline README.md:8]:
```
/home/maxime/Project/Epitech/2020-2021/T-AIA_901/venv/lib/python3.7/site-packages/pydub/utils.py:170: RuntimeWarning: Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work
  warn("Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work", RuntimeWarning)
loading : fr_core_news_sm
.loading : fr_core_news_sm
Floading : fr_core_news_sm
F...
======================================================================
FAIL: test_isModelCreated (naturalLanguageProcessing.TestNlp.TestNLP)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/maxime/Project/Epitech/2020-2021/T-AIA_901/naturalLanguageProcessing/TestNlp.py", line 36, in test_isModelCreated
    self.assertEqual(True, NLP.isModelCreated())
AssertionError: True != False

======================================================================
FAIL: test_predict (naturalLanguageProcessing.TestNlp.TestNLP)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/maxime/Project/Epitech/2020-2021/T-AIA_901/naturalLanguageProcessing/TestNlp.py", line 30, in test_predict
    self.assertEqual(NLP.predict(text), result)
naturalLanguageProcessing.BadPhraseException.BadPhraseException: ('No End or invalid instruction', 'Bad Phrase')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/maxime/Project/Epitech/2020-2021/T-AIA_901/naturalLanguageProcessing/TestNlp.py", line 32, in test_predict
    self.assertEqual(e.message, result)
AssertionError: 'Bad Phrase' != ('Lille', 'Paris')

----------------------------------------------------------------------
Ran 6 tests in 7.310s

FAILED (failures=2)
```

Comment (maximegavens) [reply README.md:8]: j'ai les deux tests qui échouent ??

**Review by maximegavens — APPROVED** [review]

**Review by jeanEpitech — APPROVED** [review]
It's workin'

Good job Billy Boy!

**Review by jeanEpitech — APPROVED** [review]

**Pull request merged**